src/components/navigation-header/components/product-page-content/utils/__tests__/get-nav-items.test.ts

```diff
@@ -32,25 +32,16 @@ describe('getNavItems', () => {
 		    "url": "/waypoint",
 		  },
 		  Object {
-		    "iconColorTheme": "waypoint",
-		    "items": Array [
-		      Object {
-		        "icon": "docs",
-		        "label": "General Documentation",
-		        "path": "/waypoint/docs",
-		      },
-		      Object {
-		        "icon": "terminal-screen",
-		        "label": "CLI",
-		        "path": "/waypoint/commands",
-		      },
-		      Object {
-		        "icon": "plug",
-		        "label": "Plugins",
-		        "path": "/waypoint/plugins",
-		      },
-		    ],
-		    "label": "Documentation",
+		    "label": "General Documentation",
```

Comment:
💭 We should update the test data too to not use `General Documentation`

Reply:
I dig it - updated 👍 ✅